src/lava/lib/peripherals/realsense/realsense.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+import numpy as np
+import os
+import cv2
+
+import pyrealsense2 as rs
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import OutPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+
+
+class DirectRealsenseInput(AbstractProcess):
+    """
+    Outputting frames of RGB and Depth from Realsense camera
+    fetched from Realsense SDK or from recording files.
```

Comment:
Please add a short description of the parameters for this class according to our coding conventions (https://lava-nc.org/developer_guide.html#docstring-format).

Reply:
added description of the parameters